react-events-and-state/src/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  render() {
+    if (this.state.isClicked) {
+      return <button onClick={this.handleClick}>Thanks!</button>;
+    }
+    return <button onClick={this.handleClick}>Click Me!</button>;
+
+  }
+
+  handleClick(e) {
+    e.preventDefault();
```

Comment:
You only need prevent default on a form's submit handler, here it isn't necessary.

Reply:
I think I might've accidentally wrote that when I was looking at the docs and they were using onSubmit for the example